src/components/cc-feature-list/cc-feature-list.types.d.ts

```diff
@@ -0,0 +1,27 @@
+export type FeatureListState = FeatureListStateLoaded | FeatureListStateLoading | FeatureListStateError;
+
+type FeatureStatus = 'alpha' | 'beta' | 'preview' | '';
+
+export interface Feature {
+  id: string;
+  name: string;
+  description: string;
+  options: { label: string; value: string }[];
+  value: string;
+  status: FeatureStatus;
+  documentationLink: string;
+  feedbackLink: string;
```

Comment:
nitpick: having a feedback link seems optional, can we reflect that in the type?